### PR TITLE
AUT-277: Add a robots.txt to the OIDC API - Build only

### PR DESCRIPTION
**Pull request by dbes-gds**
## What?

Add a robots.txt to the OIDC API - Build only.

## Why?

As this is a public API it's being scraped by robots which creates unecessary noise in the logs.
Adding a robots.txt to keep them away.

### CODE REVIEW

File: ci/terraform/oidc/api-gateway.tf [hunk 1]
```diff
@@ -133,6 +133,7 @@ resource "aws_api_gateway_deployment" "deployment" {
       module.ipv-capacity.method_trigger_value,
       var.doc_app_api_enabled ? module.doc-app-callback[0].integration_trigger_value : null,
       var.doc_app_api_enabled ? module.doc-app-callback[0].method_trigger_value : null,
+      var.use_robots_txt ? aws_api_gateway_integration_response.robots_txt_integration_response : null,
```
Comment (mrwilson) [inline ci/terraform/oidc/api-gateway.tf:136]: I think the response will need a `[0]` on it

Comment (mrwilson) [reply ci/terraform/oidc/api-gateway.tf:136]: Because there's a count

**Review by mrwilson — CHANGES_REQUESTED** [review]
Just the count change

File: ci/terraform/oidc/api-gateway.tf (hunk 1 above)
Comment (dbes-gds) [reply ci/terraform/oidc/api-gateway.tf]: thanks updated

**Review by mrwilson — APPROVED** [review]

**Pull request merged**